### PR TITLE
Update with Decidim v0.25

### PR DESCRIPTION
**Pull request by laurajaime**
- Upgrade to Decidim 0.25.2
- Bump Ruby version to 2.7.5
- Add CHANGELOG
- Fix tests

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,23 +2,18 @@
 
 source "https://rubygems.org"
 
-ruby RUBY_VERSION
```
Comment (tramuntanal) [inline Gemfile]: why?

File: Gemfile [hunk 2]
```diff
@@ -2,23 +2,18 @@
...
-
 gemspec
 
-DECIDIM_VERSION = ">= 0.17.0"
```
Comment (tramuntanal) [inline Gemfile:9]: why remove?

File: README.md [hunk 3]
```diff
@@ -42,11 +42,11 @@ sant_boi_census:
...
 ## Testing
 
-1. Run `bundle exec rake test_app`.
+Node 16.1.9 is required!
 
-2. Run tests with `bundle exec rspec`
+1. Run `bundle exec rake test_app`.
 
-5. Set the configuration values for the test app in `spec/decidim_dummy_app/config/secrets.yml`
+2. Set the configuration values for the test app in `spec/decidim_dummy_app/config/secrets.yml`
```
Comment (tramuntanal) [inline README.md:49]: Great!
Thanks for updating the README doc

File: lib/decidim/verifications/sant_boi_census/version.rb [hunk 4]
```diff
@@ -5,7 +5,7 @@ module Decidim
   module Verifications
     module SantBoiCensus
       def self.version
-        "0.0.2"
+        "0.1.2"
```
Comment (tramuntanal) [inline lib/decidim/verifications/sant_boi_census/version.rb]:
```suggestion
        "0.1.0"
```

**Review by tramuntanal — COMMENTED** [review]
Maybe you have removed too much code? :smile: 
Can you check please?

File: Gemfile (hunk 1 above)
Comment (laurajaime) [reply Gemfile]: I'll recover it!

File: Gemfile (hunk 2 above)
Comment (laurajaime) [reply Gemfile:9]: Why it's required here? It's in gemspec, isn't?

File: lib/decidim/verifications/sant_boi_census/version.rb (hunk 4 above)
Comment (laurajaime) [reply lib/decidim/verifications/sant_boi_census/version.rb]: Ups!

**Comment by laurajaime** [comment]
@tramuntanal can you review it again, please? :)

File: Gemfile (hunk 2 above)
Comment (tramuntanal) [reply Gemfile:9]: The idea is to force the min version when declaring `gem "decidim"`

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**